index.html

```diff
@@ -0,0 +1,281 @@
+
+          <ul class="works">
+            <li>
+              <img src="./Images/icon-frameworks.png" alt="Frameworks" />
+            </li>
+            <li class="frameworks">Frameworks</li>
+            <li>
+              <ul class="language">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+              </ul>
+            </li>
+          </ul>
+
+          <ul class="works">
+            <li>
+              <img src="./Images/icon-skills.png" alt="Skills" />
+            </li>
+            <li class="skills">Skills</li>
+            <li>
+              <ul class="language">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="contact">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+
+        <form action="https://formspree.io/f/mzbwnlbg" method="post">
+          <label for="fname"></label>
+          <input
+            type="text"
+            name="fname"
+            id="fname"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          /><br />
+          <label for="email"></label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email address"
+            required
+          /><br />
+          <textarea
+            name="message"
+            id="type"
+            cols="40"
+            rows="7"
+            maxlength="500"
+            placeholder="Enter Text Here"
+            required
+          ></textarea>
+          <br />
+          <input type="submit" value="Get in touch" class="button" />
+        </form>
+      </section>
+    </main>
+
+    <footer>
+      <ul class="social">
+        <li>
+          <a href="https://github.com/Mike47ip"
+            ><ion-icon name="logo-github"></ion-icon
+          ></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/michael-darkwah-81a039141/"
+            ><ion-icon name="logo-linkedin"></ion-icon
+          ></a>
+        </li>
+        <li>
+          <a href="https://twitter.com/michael-darkwah"
+            ><ion-icon name="logo-twitter"></ion-icon
+          ></a>
+        </li>
+        <li>
+          <a href="https://web.facebook.com/michael-darkwah"
+            ><ion-icon name="logo-facebook"></ion-icon
+          ></a>
+        </li>
+      </ul>
+
+      <a href="mailto:agyemankojo47@gmail.com">agyemankojo47@gmail.com</a>
+    </footer>
+  </body>
+
+  <script
+    type="module"
+    src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
+  ></script>
```

Comment:
- [ ] [OPTINAL] As stated by the previous reviewer It will be better 😃 if Scripts can be placed at the end of your <body> or in the <head> section of an HTML page, or in both. Kindly fix that.💯🚀